packages/sysweb3-keyring/src/eth-manager.ts

```diff
@@ -186,61 +199,53 @@ export const Web3Accounts = (): IWeb3Accounts => {
 
     const tokensTransfers: any = [];
 
-    try {
-      const { chainId, label, apiUrl, url } = network;
+    const { chainId, label, apiUrl, url } = network;
```

Comment:
Why has the try here been removed? 
There are some network calls that might fail and we need to notify clients of it with the error only and not by breaking their code execution but if there is a good reason it's fine too.